test/unit/test-extension-analytics.js

```diff
@@ -29,6 +29,8 @@ import {
 import {macroTask} from '../../testing/yield';
 import {registerElement} from '../../src/service/custom-element-registry';
 
+/* eslint react-hooks/rules-of-hooks: "off" */
```

Comment:
How is this different from `/* eslint-disable react-hooks/rules-of-hooks */` (the form used everywhere else in the codebase)?

Reply:
Updated.